imports/server/methods/insertDocumentImage.ts

```diff
@@ -0,0 +1,87 @@
+import { check, Match } from 'meteor/check';
+import { fetch } from 'meteor/fetch';
+import { Meteor } from 'meteor/meteor';
+import Documents from '../../lib/models/Documents';
+import Settings from '../../lib/models/Settings';
+import insertDocumentImage from '../../methods/insertDocumentImage';
+
+insertDocumentImage.define({
+  validate(arg) {
+    check(arg, {
+      documentId: String,
+      sheetId: Number,
+      image: Match.OneOf({
+        source: Match.OneOf('upload'),
+        filename: String,
+        contents: String,
```

Comment:
Mind leaving a comment indicating the encoding here?

Reply:
There's a comment in `imports/methods/insertDocumentImage.ts`, although I'll expand it to note that it's specifically a data URL